### PR TITLE
ibmcloud: remove depricated terraform folder/scripts

### PR DESCRIPTION
**Pull request by Jordan9500**
Content of commit
Remove the entire directory for the terraform scripts

Fixes: #1039

### CODE REVIEW

**Review by stevenhorsman — APPROVED** [review]
LGTM, but lets be cautious and wait until after code freeze to merge this. Thanks!

**Review by jtumber-ibm — APPROVED** [review]

**Pull request merged**
